app/src/main/java/com/epfl/neighborfood/neighborfoodandroid/ui/activities/VendorProfileActivity.java

```diff
@@ -42,6 +49,7 @@ public void onClick(View v) {
         switch (v.getId()) {
             case R.id.SubscribeId:
                 subscribed = !subscribed;
+                vmodel.subscribeToVendor("JDpBLvLCTefqXx03riPH9072KYu2").addOnCompleteListener((a)-> System.out.println("Subscribed"));

```

Comment:
We should change this to the notfication button instead

Reply:
we should add unsubscribtion too. The unsubscribtion function is already there.

Reply:
yes, I thought this will come with the feature of linking the vendoractivity with the database,